kf_lib_data_ingest/common/concept_schema.py

```diff
@@ -102,10 +108,19 @@ class DIAGNOSIS(PropertyMixin):
         CATEGORY = None
         UBERON_TUMOR_LOCATION_ID = None
         EVENT_AGE_DAYS = None
+        VERIFICATION = None
+
+        class ABATEMENT_EVENT_AGE(QuantityMixin):
+            pass
 
         class EVENT_AGE(QuantityMixin):
             pass
 
+        class ANATOMY_SITE:
+            NAME = None
+            NCIT_ID = None
+            UBERON_ID = None
+
```

Comment:
I think we don't need `ANATOMY_SITE.NCIT_ID`. I'll remove that.

In general, I've taken the approach of keeping old fields and adding new fields in an effort to ease the transition from our current Dataservice model to a future model (whether a modification of Dataservice or a new FHIR service). Then in the Data Tracker extract configs, I tried to map a template column to both an "old"/current concept (because those are used in the current target service plugin) and the new concept (so that the extracted files will have the new fields that can be used by a future target service plugin).  Not sure if that's the best approach but that's why I have `ANATOMY_SITE` concepts and the other location concepts